terraform/a2ha-terraform/modules/automate/templates/provision.sh.tpl

```diff
@@ -31,7 +31,8 @@ export -f truncate_with_timestamp
 
 save_space() {
   # this will truncate all but the most recent frontend .aib files
-  find ${tmp_path} -name frontend\*aib -printf "%T+\t%p\n" | sort -r | awk '{print $2}' | tail -n +2 | xargs -n1 -I{} bash -c 'truncate_with_timestamp "$@"' _ {}
+ # find /var/tmp/ -name frontend\*aib -printf "%T+\t%p\n" | sort -r | awk '{print $2}' | tail -n +2 | xargs -n1 -I{} bash -c 'truncate_with_timestamp "$@"' _ {}
```

Comment:
If there is any specific reason for having this as a comment, please also mention that reason. Otherwise, can we delete this commented code?

Reply:
removed the comments